sdk/core/azure-core-amqp/inc/azure/core/amqp/internal/link.hpp

```diff
@@ -37,6 +37,7 @@ namespace Azure { namespace Core { namespace Amqp { namespace _internal {
 namespace Azure { namespace Core { namespace Amqp { namespace _detail {
   class LinkImpl;
 
+#if ENABLE_UAMQP
```

Comment:
Why is this enum class only available when `ENABLE_UAMQP` is on? Probably deserves a comment to explain the rationale.

Reply:
The enum class is a UAMQP construct. It should never be available unless UAMQP is enabled.